results/cmd/api/main.go

```diff
@@ -161,3 +172,79 @@ func (s *server) DeleteTaskRunResult(ctx context.Context, req *pb.DeleteTaskRunR
 	}
 	return nil, nil
 }
+
+// ListTaskRunsResult receives a ListTaskRunRequest from users and return to users a list of TaskRuns according to query
+func (s *server) ListTaskRunsResult(ctx context.Context, req *pb.ListTaskRunsRequest) (*pb.ListTaskRunsResponse, error) {
+	env, err := cel.NewEnv(
+		cel.Types(&pb.TaskRun{}),
+		cel.Declarations(decls.NewIdent("taskrun", decls.NewObjectType("tekton.TaskRun"), nil)),
+	)
```

Comment:
This is always going to be the same for the server, regardless of the request, right? We should pull this into a `New()` func and set it once on server creation.

Reply:
done

Reply:
I don't think this was addressed. This would help reduce some between main.go and main_test.go! :)